packages/manager/apps/pci-instances/src/pages/instances/delete/DeleteInstance.page.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC } from 'react';
+import { useNavigate, useParams, useSearchParams } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { useNotifications } from '@ovh-ux/manager-react-components';
+import { useDeleteInstance } from '@/data/hooks/instance/useDeleteInstance';
+import { updateDeletedInstanceStatus } from '@/data/hooks/instance/useInstances';
+import queryClient from '@/queryClient';
+import { DeleteModal } from './modal/DeleteModal.component';
+
+const DeleteInstance: FC = () => {
+  const { t } = useTranslation(['delete', 'common']);
+  const navigate = useNavigate();
+  const { projectId } = useParams() as { projectId: string };
+  const [searchParams] = useSearchParams();
+  const instanceId = searchParams.get('instanceId');
+  const instanceName = searchParams.get('instanceName');
+  const { addError, addSuccess } = useNotifications();
+
+  const handleModalClose = () => {
+    navigate('..');
+  };
+
+  const handleMutationSuccess = () => {
+    if (instanceId) {
+      updateDeletedInstanceStatus(queryClient, instanceId);
+    }
+    addSuccess(
```

Comment:
if we don't have an instanceId we still put a notification ?

Reply:
instancsId is always defined when method is called in render cycle, but yep, I can include it in the if statement 👍

Reply:
If the call isn't possible you can add "!" at the end or put it in a casted variable

Reply:
I removed the `if` statement and better handled the `undefined` parameter